src/sass/Fabric.scss

```diff
@@ -12,8 +12,10 @@
 @import './Animation';
 @import './BrandIcon';
 @import './Color';
+@import './Font.Definitions';
 @import './Font';
 @import './Grid';
+@import './IconFont';
```

Comment:
Ah yes, good catch--I'll change this. And I agree, I'd *really* love to stick with single-word filenames as much as possible, but not sure how else to split this out.